src/main/java/com/revature/Barbee/utils/ResultSetPrinter.java

```diff
@@ -0,0 +1,57 @@
+package com.revature.Barbee.utils;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ResultSetPrinter {
+
+    /**
+     * Harvests all the data from a ResultSet required to print as a table, with tab-aligned columns.
+     * 
+     * @param rs {@code ResultSet} to be printed
+     * @throws SQLException if a database access error occurs or a closed {@code ResultSet} is provided
+     * @see com.revature.Barbee.utils.UltimateStringTabber#tabColumn UltimateStringTabber tabColumn
+     */
+    public static void printResultSet(ResultSet rs) throws SQLException {
+
+        // Get the column names (and an easier way to see how many columns there are)
+        List<String> columnNames = new ArrayList<>();
+        for (int i = 0; i < rs.getMetaData().getColumnCount(); i++) {
+            columnNames.add(rs.getMetaData().getColumnName(i+1));
+        }
+
+        // Prepare to store all the data in a column list
+        List<List<String>> dataColumns = new ArrayList<>();
+        for (int i = 0; i < columnNames.size(); i++) {
+            dataColumns.add(new ArrayList<>());
+            dataColumns.get(i).add(columnNames.get(i));
+        }
```

Comment:
Another way of doing this is with an enhanced for loop and initialization of the array:
```
for (String name : columnNames) {
    dataColumns.add(new ArrayList<String>() {{ add(name); }} );
}
```